index.html

```diff
@@ -4,63 +4,211 @@
-              <img class="carousel-food-spin__img" src="image/food2.png" alt="dish 2">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish 3">
-              <img class="carousel-food-spin__img" src="image/food3.png" alt="dish 3">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish main">
-              <img class="carousel-food-spin__img" src="image/foodmain.png" alt="dish main">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish 4">
-              <img class="carousel-food-spin__img" src="image/food4.png" alt="dish 4">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish 5">
-              <img class="carousel-food-spin__img" src="image/food5.png" alt="dish 5">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish 6">
-              <img class="carousel-food-spin__img" src="image/food1.png" alt="dish 6">
-            </a>
-          </li>
-          <li class="carousel-food-spin__dish js-carousel-food-spin-dish">
-            <a href="" aria-label="click to see the menus of dish 7">
-              <img class="carousel-food-spin__img" src="image/food2.png" alt="dish 7">
-            </a>
-          </li>
-          <li class="carousel-food-spin__controls js-carousel-foord-spin-dish">
-            <button class="carousel-food-spin__control js-prev" aria-label="see plates on the right">‹</button>
-            <button class="carousel-food-spin__control js-next" aria-label="see plates on the left">›</button>
-          </li>
-        </ul>
+    <section class="food-spin__info" aria-label="menu of available dishes">
+      <div class="food-spin__ellipse js-food-spin-ellipse" aria-hidden="true"></div>
+      <div class="food-spin__let" aria-label="information about the dishes">
+        <p class="food-spin__price js-food-spin-price" aria-label="price the dishes">$32</p>
+        <dl>
+          <dt class="food-spin__title js-food-spin-title">Green Goddess Chicken Salad</dt>
+          <dd class="food-spin__description js-food-spin-description">It Is A Non Vegetarian Salad Which Consists Of The Green Goddess Dressing Mixed With Chicken, Peppers, Olives And Celery.</dd>
+        </dl>
+        <button class="food-spin__order-button js-food-spin-order-button" aria-label="place your order now">ORDER NOW</button>
+        <p class="food-spin__video" aria-hidden="true">Watch the Video</p>
```

Comment:
Si veo que algo dice 'mira el video', esperaria un link al video
```suggestion
        <a href="#" class="food-spin__video" aria-hidden="true">Watch the Video</a>
```